source/guide_grafana-loki.rst

```diff
@@ -0,0 +1,277 @@
+.. highlight:: console
+
+.. author:: Franz Wimmer <https://codefoundry.de>
+
+.. tag:: lang-go
+.. tag:: monitoring
+.. tag:: logs
+.. tag:: logshipper
+.. tag:: observability
+.. tag:: audience-admins
+
+.. sidebar:: About
+
+  .. image:: _static/images/loki.png
+      :align: center
+
+############
+Grafana Loki
+############
+
+.. tag_list::
+
+Loki is a log aggregation system designed to store and query logs from all your applications and infrastructure.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :lab:`Grafana <guide_grafana>`
+
+License
+=======
+
+Loki is licensed under the GNU Affero General Public License (AGPL).
+
+All relevant legal information can be found here
+
+  * https://github.com/grafana/loki/blob/main/LICENSE
+
+Prerequisites
+=============
+
+We need to prepare a couple of directories and create a database which is used by grafana.
+
+Directory for building loki from source:
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/software
+ [isabell@stardust ~]$
+
+Directory for storing the custom configuration files:
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/etc/loki
+ [isabell@stardust ~]$
+
+Directory for storing the log files:
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/var/log/grafana
+ [isabell@stardust ~]$
+
+Loki Installation
+=================
+
+Clone loki from the official git repository:
+
+::
+
+ [isabell@stardust ~]$ git clone https://github.com/grafana/loki ~/software/
+ [isabell@stardust ~]$ cd ~/software/loki
+ [isabell@stardust loki]$
+
+Build loki
+----------
+
+::
+
+ [isabell@stardust loki]$ make loki
```

Comment:
provide output:

```

CGO_ENABLED=0 go build -ldflags "-extldflags \"-static\" -s -w -X github.com/grafana/loki/pkg/util/build.Branch=main -X github.com/grafana/loki/pkg/util/build.Version=main-05d2110 -X github.com/grafana/loki/pkg/util/build.Revision=05d211068 -X github.com/grafana/loki/pkg/util/build.BuildUser=isabell@stardust.uberspace.de -X github.com/grafana/loki/pkg/util/build.BuildDate=2022-12-14T22:06:01Z" -tags netgo -o cmd/loki/loki ./cmd/loki
```

and state that this will take some time